tonic/models/vic/vic.py

```diff
@@ -67,8 +67,18 @@ def run(self, global_param, logdir=None):
 
         return self.returncode
 
-    def _call_vic(self, *args):
-        vic_args = [self.executable] + [a for a in args]
+    def _call_vic(self, *args, mproc_num=None):
+
+	vic_args = []
+
+	if mproc_num is not None:
+		if  type(mproc_num) != int:  
+			raise AssertionError 
+			print("number of processors has not been specified as an integer") 
+ 
+		vic_args.extend = "mpirun -np %s" % str(mproc_num)  
```

Comment:
rather than letting `str` pick your string format, how about

```Python
vic_args.extend(['mpirun', '-np',  '{0:d}'.format(mpi_proc)])
```

Note that you also have a sytax error in `vic_args.extend`.